_i18n/de/resources/moneropedia/wallet.md

```diff
@@ -1,18 +1,17 @@
+### Grundlagen
 
-A Monero account, or wallet, stores the information necessary to send and receive Moneroj.  In addition to sending and receiving, the Monero Wallet software keeps a private history of your transactions and allows you to cryptographically sign messages.  It also includes Monero mining software and an address book.
+Ein Monero-Konto beziehungsweise ein Wallet speichert die Informationen, die zum Senden und Empfangen von Monero benötigt werden. Zusätzlich zum Senden und Empfangen führt und bewahrt die Wallet-Software von Monero eine private Chronik deiner Transaktionen und ermöglicht es dir zudem, Nachrichten kryptografisch zu signieren. Sie enthält außerdem ein Programm zum Minen von Monero und ein Adressbuch.
 
-The term "hot wallet" describes a Monero @account which is connected to the Internet.  You can send funds easily but security is much lower than a cold wallet.  Never store large amounts of cryptocurrency in a hot wallet!
+Ein Spar-Wallet ("Hot-Wallet") ist ein mit dem Internet verbundenes Monero-@Konto. Du kannst Gelder zwar bequem versenden, die Sicherheit ist allerdings sehr viel niedriger als es bei einem Spar-Wallet der Fall ist. Bewahre niemals große Mengen von Kryptowährung in einem Zahlungs-Wallet auf! 
 
-A cold wallet is generated on a trusted device or computer via an @airgap.  If the device is to be reused, the data storage should be securely overwritten.  As soon as a cold wallet is connected to the Internet or its mnemonic phrase or @spend-key is entered on an Internet-connected device, it's no longer "cold" and should be considered "hot".
+Ein Spar-Wallet ("Cold-Wallet") wird durch einen @Airgap auf einem vertrauten Gerät oder Computer erzeugt. Falls das Gerät später wiederverwendet werden soll, sollte der Datenspeicher davor sicher überschrieben werden. Sobald ein Spar-Wallet mit dem Internet verbunden wird oder seine mnemonische Passphrase oder sein @Spend-Key in ein mit dem Internet verbundenes Gerät eingegeben wird, sollte es als Zahlungs-Wallet betrachtet werden.
 
-A Monero @paper-wallet can be generated by downloading the source code of https://moneroaddress.org/.  Verify the signature of the code on a trusted airgapped device.  Create the wallet and print or store it on the media of your choice.
-
-Monero accounts and paper-wallets can be stored on any media - paper, USB drive, CD/DVD, or a hardware wallet device (Ledger available since June 2018).
+Ein Monero-@Paper-Wallet kann erzeugt werden, indem der Quellcode von https://moneroaddress.org/ heruntergeladen wird. Die Signatur des Codes sollte auf einem vertrauten, mittels @Airgap isolierten Gerät verifiziert werden. Danach kann das Wallet erstellt und ausgedruckt oder auf einem ausgewählten Datenträger gespeichert werden.
 
+Monero-Konten und Paper-Wallets können auf jeglichen Medien gesichert werden: Papier, USB-Stick, CD/DVD oder einem Hardware-Wallet-Gerät (Ledger seit Juni 2018 erhältlich).
```

Comment:
I think with this we're through all the pull requests. :) Thanks for the work you put into translating this. Let me now if you need anything else.

Reply:
@stmax82 thank you very much for your time, it's really appreciated :)

Reply:
@stmax82 Thanks a lot for your whole review and the time you "sacrificed" for it! If you have fun doing this: I'm going to push the translations of the user guides very soon. :)